sample/app/build.gradle

```diff
@@ -28,6 +28,9 @@ buildProperties {
     env {
         from System.getenv()
     }
+    cli {
+        from project.properties
```

Comment:
Does `from project.properties` include multiple sources of properties? e.g. CLI or gradle.properties